<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Q1: capital of pakistan??</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>karachi</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>islamabad</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>peshawar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>khyber pakhtunkhwa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
